src/VisualStudio/CSharp/Impl/project.json

```diff
@@ -1,39 +1,39 @@
 {
     "dependencies": {
         "Microsoft.VisualStudio.Editor": {
-            "version": "14.3.25407",
+            "version": "15.0.25726-Preview5",
             "suppressParent": "all"
         },
         "Microsoft.VisualStudio.Language.Intellisense": {
-            "version": "14.3.25407",
+            "version": "15.0.25726-Preview5",
             "suppressParent": "all"
         },
         "RoslynDependencies.Microsoft.VisualStudio.GraphModel": {
-            "version": "14.3.25407",
+            "version": "15.0.25726-Preview5",
             "suppressParent": "all"
         },
-        "Microsoft.VisualStudio.Shell.14.0": {
-            "version": "14.3.25407",
+        "Microsoft.VisualStudio.Shell.15.0": {
+            "version": "15.0.25726-Preview5",
             "suppressParent": "all"
         },
         "Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime": {
-            "version": "14.3.25407",
+            "version": "15.0.25726-Preview5",
             "suppressParent": "all"
         },
         "RoslynDependencies.Microsoft.Internal.VisualStudio.Shell.Interop.14.0.DesignTime": {
             "version": "14.3.25407",
```

Comment:
Possibly. To the extent that we're only using this to pick up some COM interfaces it shouldn't matter.